cluster/gce/configure-vm.sh

```diff
@@ -710,8 +710,7 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
```

Comment:
it looks like this should already be populated with a cert generated with the correct group. what are you seeing?